frontend/portals/MaintenanceMode/index.jsx

```diff
@@ -62,15 +76,45 @@ class MaintenanceMode extends Component {
     openPageExtern({ src: imageHref });
   };
 
+  /**
+   * Checks if the app version is blocked.
+   * @param {string} appVersion App version
+   * @returns {bollean}
+   */
+  appVersionIsBlocked = (appVersion) => {
+    let appVersions = [];
+
+    if (this.props.isIosDevice) {
+      appVersions = iosAppVersions.replace(/ /g, '').split(',');
+    } else {
+      appVersions = androidAppVersions.replace(/ /g, '').split(',');
+    }
+
+    // All versions
+    if (appVersions.includes('')) {
```

Comment:
This is is a weird test. Can you explain what you are trying to do? I bet there is a better way.
Even without the explanation I think it would be better to just make the iosAppVersions and androidAppVersions configurations arrays rather than strings then you don't have  clean them up, and turn them into arrays.

you can then quickly assign the appVersions value with a quick ternary statement like 
```
const appVersions = this.props.isIosDevice ? iosAppVersions : androidAppVersions;
``` 

if you want to return true if there are no appVersions defined in the configuration all you have to do is test the length property of the array like
```
if (!appVersions.length) {
    return true
}
```

Reply:
No explanation needed, you are right!